### ISSUE TITLE
Composition editor starcounter-include dropdown should allow to filter by View URI

### ISSUE THREAD

**Issue by warpech** [Starcounter/Blending#286]
Currently, the dropdown only allows to filter by CSS selector of `starcounter-include`, which is not very useful. Filtering by typing an app name or HTML file name would be more useful.

**Comment by tomalec** [comment]
WDYT about not showing the selector at all, as I think that was one of the reasons some of Apps Team members started calling starcounter-include "slots", what in my opinion could lead to lots of confusion.

Then user will be able to pick a blending point by:
1. Picking the ViewURI it includes (significant part of its id),
2. Point and click - #22,
3. Typing selector by hand, which could even be `body>nth-child(4)>div>.foo`

**Comment by alshakero** [comment]
Sorry I didn't see @tomalec comment and did the whole search thing and now it's looking very beautiful. See https://github.com/Starcounter/Blending/pull/378

**Comment by tomalec** [comment]
I think my proposition would actually made you PR waaay more simpler, as it changes view to be searchable name, and not use selector at all, as I no longer think it anyhow readable and intuitive for humans.

**Comment by alshakero** [comment]
> would actually made you PR waaay more simpler

That's correct. But now it's very nice and smooth to use. Even if we remove the selector the highlighting is such a breeze. Try it if you want.
